mu-plugins/blocks/global-header-footer/header.php

```diff
@@ -13,44 +13,108 @@
-
-<!-- wp:navigation-link {"label":"Themes","url":"https://wordpress.org/themes/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Patterns","url":"https://wordpress.org/patterns/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"OpenVerse","url":"https://wordpress.org/openverse/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Learn","url":"https://learn.wordpress.org/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Support","url":"https://wordpress.org/support/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"News","url":"https://wordpress.org/news","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"About","url":"https://wordpress.org/about/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Get Involved","url":"https://make.wordpress.org/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Showcase","url":"https://wordpress.org/showcase/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Mobile","url":"https://wordpress.org/mobile/","kind":"custom","isTopLevelLink":true} /-->
-
-<!-- wp:navigation-link {"label":"Hosting","url":"https://wordpress.org/hosting/","kind":"custom","isTopLevelLink":true} /-->
-<!-- /wp:navigation -->
-
-<!-- wp:buttons {"className":"site-header__action-buttons"} -->
-<div class="wp-block-buttons site-header__action-buttons"><!-- wp:button {"className":"site-header__search"} -->
-<div class="wp-block-button site-header__search"><a class="wp-block-button__link" href="https://wordpress.org/search/">Search</a></div>
-<!-- /wp:button -->
-
-<!-- wp:button {"className":"site-header__get_wordpress"} -->
-<div class="wp-block-button site-header__get_wordpress"><a class="wp-block-button__link" href="https://wordpress.org/download/">Get WordPress</a></div>
-<!-- /wp:button --></div>
-<!-- /wp:buttons --></header>
-<!-- /wp:group -->
+<!-- wp:group {"tagName":"header","className":"site-header"} -->
+<header class="wp-block-group site-header">
+	<!-- wp:image {"width":27,"height":27,"className":"site-header__wporg-logo-mark"} -->
+	<figure class="wp-block-image is-resized site-header__wporg-logo-mark">
+		<a href="https://wordpress.org/">
+			<img src="https://wordpress.org/style/images/w-mark.svg" alt="" width="27" height="27" />
+		</a>
+	</figure>
+	<!-- /wp:image -->
+
+	<!-- wp:image {"width":160,"height":24,"className":"site-header__wporg-logo-full"} -->
+	<figure class="wp-block-image is-resized site-header__wporg-logo-full">
+		<a href="https://wordpress.org/">
+			<img src="https://wordpress.org/style/images/wporg-logo.svg?3" alt="" width="160" height="24" />
+		</a>
+	</figure>
+	<!-- /wp:image -->
+
+	<!-- wp:group {"tagName":"div","className":"site-header__search-container"} -->
```

Comment:
I think you can omit `"tagName":"div"` since that's the default tag for a group block.

Reply:
👍🏻